pkg/services/vmoperator/resource_policy.go

```diff
@@ -106,3 +122,32 @@ func (s *RPService) createVirtualMachineSetResourcePolicy(ctx context.Context, c
 	}
 	return vmResourcePolicy, nil
 }
+
+func getClusterModuleGroups(ctx context.Context, ctrlClient client.Client, cluster *clusterv1.Cluster) ([]string, error) {
+	machineDeploymentNames, err := getMachineDeploymentNamesForCluster(ctx, ctrlClient, cluster)
+	if err != nil {
+		return nil, err
+	}
+
+	clusterModuleGroups := append([]string{ControlPlaneVMClusterModuleGroupName}, machineDeploymentNames...)
```

Comment:
Might be we should add also fmt.Sprintf("%s-workers-0", cluster.Name) if not already there, at least for the transition time